app/views/sign_up.erb

```diff
@@ -0,0 +1,29 @@
+<% if flash[:errors] && !flash[:errors].empty? %>
```

Comment:
Good thing to look at: 
https://github.com/makersacademy/chitter-challenge/blob/master/docs/review.md#use-partials-with-standard-html-conventions